assets/src/apps/delivery/store/features/page/actions/loadInitialPageState.ts

```diff
@@ -173,14 +172,6 @@ export const loadInitialPageState = createAsyncThunk(
           }, 0);
           resumeSequenceId = sequence[resumeTarget].custom.sequenceId;
         }
-        // need to check the visitHistory to see if the resumeSequenceId is in there and is NOT the latest, then we need to set history mode to true
```

Comment:
Previously, we used to check the visitHistory to see if the  resumeSequenceId is in there and if we found it then we used to set history mode to true.  

We don't need to that anymore because when we resume now, we always navigated to the current active screen.